client/src/featureform/register.py

```diff
@@ -1086,10 +1087,13 @@ def df_transformation(self,
         Returns:
             decorator (DFTransformationDecorator): decorator
         """
+
         if not isinstance(owner, str):
             owner = owner.name()
         if owner == "":
             owner = self.must_get_default_owner()
+        if variant == "":
```

Comment:
why can't we default it to `"default"` instead of empty string? Is it to be consistent?